app/components/chat.tsx

```diff
@@ -459,6 +459,10 @@ export function ChatActions(props: {
     if (!show) {
       props.setAttachImages([]);
       props.setUploading(false);
+    } else {
+      // 为visionModel时不附带历史消息
+      const newModelConfig = chatStore.currentSession().mask.modelConfig;
```

Comment:
It might effect gpt vision model, it should be modified in google api provider. And plz add comments in English.